client/my-sites/comments/controller.js

```diff
@@ -12,27 +12,57 @@ import route from 'lib/route';
 import CommentsManagement from './main';
 import controller from 'my-sites/controller';
 
-export const comments = function( context ) {
-	const siteSlug = route.getSiteFragment( context.path );
-	const status = context.params.status === 'pending' ? 'unapproved' : context.params.status;
+export const isStatusValid = status => {
+	const validStatuses = [ 'pending', 'approved', 'spam', 'trash', 'all' ];
+	return validStatuses.indexOf( status ) !== -1;
```

Comment:
I realize I tend to avoid lodash because I've never really used it :) I like your justification, though – it _does_ make it easier to understand. Done in 509c2f40a2a24bf2708313efdd05a29dd8633170.